lib/util/equals.js

```diff
@@ -0,0 +1,31 @@
+function equals(left, right) {
```

Comment:
This should eventually go in a math-ast tools package.  This function though could be replaced with any function that does deep comparison of two objects.